### PR TITLE
[fix](doc) Added english translation for monitoring Metric description page

### PR DESCRIPTION
**Pull request by Nitin-Kashyap**
## Proposed changes
Added english translation for monitoring Metric description page

### CODE REVIEW

**Review by luzhijing — DISMISSED** [review]
LGTM

**Comment by luzhijing** [comment]
run buildall

**Review by httpshirley — APPROVED** [review]

**Comment by httpshirley** [comment]
run builall

**Comment by httpshirley** [comment]
run buildall

**Review by httpshirley — APPROVED** [review]

File: docs/en/docs/admin-manual/maint-monitor/monitor-metrics/metrics.md [hunk 1]
```diff
@@ -26,5 +26,317 @@ under the License.
...
+
+## Monitoring levels and best practices
+
+**The last column in the table marks the importance level of the monitoring items. P0 means the most important, the larger the value , the lower the importance.**
+
+The vast majority of monitoring metric types are Counter . That is the cumulative value. You can obtain effective information by collecting monitoring values at intervals (such as every 15 seconds) and calculating the slope per unit time .
+
+The query error rate can be obtained by calculating the slope of `doris_fe_query_err`, presented as (error per second).
+
+> 欢迎完善此表格以提供更全面有效的监控指标。
```
Comment (luzhijing) [inline docs/en/docs/admin-manual/maint-monitor/monitor-metrics/metrics.md:66]: This sentence is missing translation.

**Comment by yiguolei** [comment]
@Nitin-Kashyap  testcase failed

**Comment by Nitin-Kashyap** [comment]
run buildall

**Review by morningman — APPROVED** [review]
LGTM

**Pull request merged**